src/main/resources/portal.properties.EXAMPLE

```diff
@@ -202,3 +202,10 @@ ucsc.build=hg19
 
 # default view in oncoprint (sample, patient (default))
 oncoprint.defaultview=patient
+
+# only set the custom annotation menu labels if we want custom annotations to appear 
+custom_annotation.binary.menu_label =
+custom_annotation.multi.menu_label =
+
+# select the custom annotations by default
+custom_annotation.default = false
```

Comment:
I think a user wants to have an option that by default, only his own passenger / driver system is selected (if present in the data). I think we can accomplish this as follows:
if custom_annotation.default == true: Disable OncoKb, Hotspots and maybe even Type